code/go/0chain.net/smartcontract/storagesc/models.go

```diff
@@ -527,7 +527,9 @@ type BlobberAllocation struct {
 	// For any case, total value of all ChallengePoolIntegralValue of all
 	// blobber of an allocation should be equal to related challenge pool
 	// balance.
-	ChallengePoolIntegralValue currency.Coin `json:"challenge_pool_integral_value"`
+	ChallengePoolIntegralValue currency.Coin     `json:"challenge_pool_integral_value"`
+	LatestCompletedChallTime   common.Timestamp  `json:"latest_complete_chall_time"`
```

Comment:
We don't need the `LatestCompletedChallTime`, it is the same value as `LatestCompletedChallenge.Created`, so add the `LatestCompletedChallenge` below is enough.